### PR TITLE
Add sig ops subprojects cluster-ops and ops-docs.

### PR DESCRIPTION
**Pull request by HumairAK**
As per: https://docs.google.com/document/d/1uEcLGP_sBSNwTLPOo_2Nu6oTqPytOTBt0c0_pVUQJwo/edit 

Notes: 
- Are subprojects names: `cluster-ops` and `ops-docs` fine?
- Added a meeting time for cluster-ops (this is the current daily sync, I suggest we consider picking a different time slow more west coast friendly)
- Does ops-docs want a meeting time? Or to be decided later?
- I didn't know where to add the OWNERS files...so I created sub dir in the sig-operations folder for now. If there is an external source added later in some other repository, we can always switch to it.

### CODE REVIEW

File: sig-operations/README.md [hunk 1]
```diff
@@ -26,16 +26,25 @@ The Chairs of the SIG run operations and processes governing the SIG.
 * Lars Kellogg-Stedman (**[@larsks](https://github.com/larsks)**), Red Hat
 
 ## Contact
-- Slack: [#operations](https://operatefirst.slack.com/messages/operations)
+- Slack: [#sig-operations](https://operatefirst.slack.com/messages/sig-operations)
 - [Mailing list](community@lists.operate-first.cloud)
 - [Open Community Issues/PRs](https://github.com/open-services-group/community/labels/sig%2Foperations)
```
Comment (larsks) [inline sig-operations/README.md:31]: Do we want to leave the community mailing list and issues repository here in the sig-operations document? I worry that people will end up creating operational issues in the community issues repository.

**Comment by larsks** [comment]
What uses `sigs.yaml` and is there any way we could use that to generate `sig-operations/README.md` instead of maintaining the same information (contacts, schedules, etc) in multiple locations?

File: sig-operations/README.md (hunk 1 above)
Comment (HumairAK) [reply sig-operations/README.md:31]: hrmm yeah this part is pulled from the templates here: https://github.com/operate-first/community/tree/main/generator 

I think we need a separate issue to revisit the generator files there. I initially pulled them from the open-services-group (which in turn probably got them from the kubernetes org). Either we replace references to osg with op1st or remove them entirely.

**Comment by HumairAK** [comment]
@larsks I generated these files using this command: 
```
podman run --rm --volume $(pwd):/workdir:Z quay.io/open-services-group/community-tooling:v0.1.0-dev
```

Which I got from @tumido . Would be nice if we can tie this in with ci/automation some how (either with prow or github actions post submit).

**Comment by HumairAK** [comment]
I also have no idea why the build is failing, it works on my local set up. Will look into it tomorrow.

File: sig-operations/README.md (hunk 1 above)
Comment (larsks) [reply sig-operations/README.md:31]: I think for now we should either remove the mailing list and issues link, or replace them with ones appropriate to sig-operations.

**Comment by HumairAK** [comment]
The netlify ci seems to be failing due to an upstream issue with jupyter-book, related to: https://github.com/executablebooks/jupyter-book/issues/1590 

Netlify uses python 3.8, and when I use their [build image](https://github.com/netlify/build-image#available-images), and do the following: 
- run their focal image as per their instructions 
- install jupyterbook on python 3.8 and 3.8.10, and try creating a fresh new jupyter-book and try to build it
- see the error: 
```
pkg_resources.ContextualVersionConflict: (click 7.1.2 (/opt/buildhome/python3.8/lib/python3.8/site-packages), Requirement.parse('click>=8.0.0'), {'black'})

The above exception was the direct cause of the following exception:
```

Not sure where to go from here. We will have to ignore the netlify failure for now as it's not related to this PR.  I don't know if we will encounter the same issue with our github action, but it doesn't hurt to try.

**Pull request merged**